Adyen/Model/Checkout/Action/IPaymentResponseAction.cs

```diff
@@ -1,4 +1,4 @@
-﻿#region License
+#region License
```

Comment:
What actually changed here?

Reply:
Nothing really. I deleted the checkout model directory bc I was testing the generation of the checkout models, but then when merging develop back in this branch it did this for some reason..

Reply:
should we include no code changes ? I think we could exclude these files